### DISCUSSION TITLE
Circular flow

### DISCUSSION THREAD

**Discussion by JasonWu008** [xyflow/xyflow#2282] [Q&A]
When the flow is circular, how can I make the edge like this
![UML 图](https://user-images.githubusercontent.com/32165408/178458575-21d64469-141f-4733-8f0f-a9347f6b4a46.jpg)

**Comment by moklick** [comment]
You need to create a [custom edge](https://reactflow.dev/docs/api/edges/custom-edges/) that implements this kind of routing behaviour.
